core/src/core-api/ux.js

```diff
@@ -0,0 +1,26 @@
+import { APP_LOADING_INDICATOR } from './../utilities/constants';
+
+/**
+ * Functions to use Luigi Core UX features.
+ * @namespace UX
+ */
+class LuigiUX {
+  /**
+   * Hides the app loading indicator.
+   * @memberof UX
+   * @since 0.6.4
+   */
+  hideAppLoadingIndicator() {
+    const appLoadingIndicator = document.querySelector(
```

Comment:
Why don't we just render the loading spinner and overlay in the place where luigi itself is rendered?

Reply:
Because Luigi itself is not rendered yet when we already show the loading spinner. We would need to create and set the spinner programmatically, which might take a while under some circumstances - we had it in place initially, but Philipp wanted to have it the way it is implemented now - simplification.

Reply:
At least we are consistent with the naming `luigi-app-*` selectors.

Reply:
👍